addon/components/mdc-tab-bar.js

```diff
@@ -12,6 +12,11 @@ const { strings } = MDCTabBarFoundation;
 export default Ember.Component.extend(MDCComponent, {
   //region Attributes
   /**
+   * @private
```

Comment:
What does the `@private` signify, given that it's within `Attributes` which are usually public?